Changes

```diff
@@ -656,6 +656,9 @@ OCaml 4.11
 - #9415: Treat `open struct` as `include struct` in toplevel
   (Leo White, review by Thomas Refis)
 
+- #9416: Avoid warning 58 in flambda ocamlnat #941
```

Comment:
The 941 at end of the line seems to be a typo.

Reply:
Fixed